utils/navigator_msgs/msg/Hosts.msg

```diff
@@ -0,0 +1 @@
+string[] hosts
```

Comment:
Thanks for the tip, this should help cut down on the code I was using to pass the dictionary.